cluster/cluster.go

```diff
@@ -43,4 +43,7 @@ type Cluster interface {
 	// FIXME: remove this method
 	// Return a random engine
 	RANDOMENGINE() (*Engine, error)
+
+	// Rename a container
+	Rename(container *Container, newName string) error
```

Comment:
This should be `RenameContainer` to be consistent with `CreateContainer`, `RemoveContainer`, ...